src/Nancy/Configuration/ConfigurationException.cs

```diff
@@ -0,0 +1,20 @@
+namespace Nancy
+{
+    using System;
+    using Nancy.Configuration;
+
+    /// <summary>
+    /// An exception related to an invalid configuration created within <see cref="INancyEnvironment"/>
+    /// </summary>
+    public class ConfigurationException : Exception
+    {
+        /// <summary>
+        /// Create an instance of <see cref="ConfigurationException"/>
+        /// </summary>
+        /// <param name="message"></param>

```

Comment:
Want the configuration object passed?

On Monday, 22 February 2016, Andreas Håkansson <notifications@github.com>
wrote:

> In src/Nancy/Configuration/ConfigurationException.cs
> <https://github.com/NancyFx/Nancy/pull/2290#discussion_r53691216>:
>
> > @@ -0,0 +1,20 @@
> > +namespace Nancy
> > +{
> > +    using System;
> > +    using Nancy.Configuration;
> > +
> > +    /// <summary>
> > +    /// An exception related to an invalid configuration created within <see cref="INancyEnvironment"/>
> > +    /// </summary>
> > +    public class ConfigurationException : Exception
> > +    {
> > +        /// <summary>
> > +        /// Create an instance of <see cref="ConfigurationException"/>
> > +        /// </summary>
> > +        /// <param name="message"></param>
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/50543/13232320/0181cb5a-d9af-11e5-9d2c-2ecef8e6d779.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2290/files#r53691216>.
>